org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/EclipseSbtBuildManager.scala

```diff
@@ -5,11 +5,13 @@ import java.io.File
 import java.lang.ref.SoftReference
 import java.util.concurrent.atomic.AtomicReference
 
+import scala.annotation.migration
```

Comment:
it doesn't look like this annotation is used. I wonder why no error is shown because of unused import.